sdlf-team/scripts/bootstrap_team.sh

```diff
@@ -42,6 +59,7 @@ if ! aws cloudformation describe-stacks --stack-name ${STACK_NAME}; then
         ParameterKey=pChildAccountId,ParameterValue="${CHILD_ACCOUNT}" \
         ParameterKey=pEnvironment,ParameterValue="${ENV}" \
         ParameterKey=pTeamName,ParameterValue="${TEAM_NAME}" \
+        ParameterKey=pSNSNotificationsEmail,ParameterValue="${NOTIFICATION_EMAIL}" \
```

Comment:
Likewise, since we might be receiving more than one SNS endpoint, we need to change this to:
`ParameterKey=pSNSNotificationsEmail,ParameterValue=\"${NOTIFICATION_EMAIL}\"`
Notice the escaped \" to parse a list instead of a string in CFN